internal/proxy/task_insert.go

```diff
@@ -134,6 +135,24 @@ func (it *insertTask) PreExecute(ctx context.Context) error {
 		return merr.WrapErrCollectionReplicateMode("insert")
 	}
 
+	collID, err := globalMetaCache.GetCollectionID(context.Background(), it.insertMsg.GetDbName(), collectionName)
+	if err != nil {
+		log.Ctx(ctx).Warn("fail to get collection id", zap.Error(err))
+		return err
+	}
+	colInfo, err := globalMetaCache.GetCollectionInfo(ctx, it.insertMsg.GetDbName(), collectionName, collID)
+	if err != nil {
+		log.Ctx(ctx).Warn("fail to get collection info", zap.Error(err))
+		return err
+	}
+	if it.schemaTimestamp != 0 {
+		if it.schemaTimestamp != colInfo.updateTimestamp {
+			err := merr.WrapErrCollectionSchemaMisMatch(collectionName)
+			log.Ctx(ctx).Warn("collection schema mismatch", zap.String("collectionName", collectionName), zap.Error(err))
```

Comment:
don't need a warn log here.
An info should be good because this error is as expected